apps/drec-ui/src/api/config.ts

```diff
@@ -1,6 +1,17 @@
 import axios, { AxiosRequestConfig } from 'axios';
 import { getAuthenticationToken } from 'shared';
 
+declare global {
+    interface Window {
+        config: {
+            REACT_APP_BACKEND_URL: string;
+            REACT_APP_GOOGLE_MAPS_API_KEY: string;
+            REACT_APP_REGISTRATION_MESSAGE_TO_SIGN: string;
+            REACT_APP_SUPPORTED_NETWORK_IDS: string;
+        };
```

Comment:
Since you are using default env variable strategy via `react-scripts` and `env.sh` you don't have to supply your env variables through the `window` object